### PR TITLE
Abdul/hid injector

### PR DESCRIPTION
**Pull request by v-abdrahman**
Created a folder for HIDInjector app & added all the project files for windows 10 Iot Enterprise.

Following changes are done:

1) file modified : HIDinjectorpackage.vcxproj
    TargetPlatformVersion from 10.0.17763.0 to 10.0.19041.0 and TargetPlatformMinVersion from 10.0.10240.0 to 10.0.15063.0.
   PlatformToolSet from v141 to v142

1) file modified : HIDinjectorTest.vcxproj
    TargetPlatformVersion from 10.0.17763.0 to 10.0.19041.0 and TargetPlatformMinVersion from 10.0.10240.0 to 10.0.15063.0.
   PlatformToolSet from v141 to v142

2) file modified : README.md
    Adding windows 10 iot Enterprise in products.
    Updated the Windows Platform from IoT Core to IoT Enterprise

### CODE REVIEW

File: HIDInjector/driver/HidInjectorKd.vcxproj [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+    <DriverType>KMDF</DriverType>
+    <PlatformToolset>WindowsKernelModeDriver10.0</PlatformToolset>
+    <ConfigurationType>Driver</ConfigurationType>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|ARM'" Label="Configuration">
+    <TargetVersion>Windows10</TargetVersion>
+    <UseDebugLibraries>True</UseDebugLibraries>
+    <DriverTargetPlatform>Universal</DriverTargetPlatform>
+    <DriverType>KMDF</DriverType>
+    <PlatformToolset>v142</PlatformToolset>
```
Comment (neilsh-msft) [inline HIDInjector/driver/HidInjectorKd.vcxproj]: Why is this set to v142 here, but for all the other platforms, it's "WindowsKernelModeDriver10.0"?

**Review by neilsh-msft — COMMENTED** [review]
I couldn't find this project in github.com/Microsoft/Windows-IotCore-Samples, but did find it in the older github.com/ms-iot/samples.  Are we bringing this sample back on purpose?

File: HIDInjector/HidinjectorPackage/HidInjectorPackage.vcxproj [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+    <UseDebugLibraries>false</UseDebugLibraries>
+    <PlatformToolset>WindowsKernelModeDriver10.0</PlatformToolset>
+    <ConfigurationType>Utility</ConfigurationType>
+    <DriverType>Package</DriverType>
+    <DisableFastUpToDateCheck>true</DisableFastUpToDateCheck>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|ARM'" Label="Configuration">
+    <TargetVersion>Windows10</TargetVersion>
+    <UseDebugLibraries>true</UseDebugLibraries>
+    <PlatformToolset>v142</PlatformToolset>
```
Comment (neilsh-msft) [inline HIDInjector/HidinjectorPackage/HidInjectorPackage.vcxproj]: Why was this changed to v142 from <PlatformToolset>WindowsKernelModeDriver10.0</PlatformToolset>?

**Review by neilsh-msft — CHANGES_REQUESTED** [review]

**Review by rsameser — APPROVED** [review]

File: HIDInjector/driver/HidInjectorKd.vcxproj (hunk 1 above)
Comment (v-abdrahman) [reply HIDInjector/driver/HidInjectorKd.vcxproj]: Updated the PlatformToolSet from v142 to KernelModelDriver10.0

File: HIDInjector/HidinjectorPackage/HidInjectorPackage.vcxproj (hunk 2 above)
Comment (v-abdrahman) [reply HIDInjector/HidinjectorPackage/HidInjectorPackage.vcxproj]: Updated the PlatformToolSet from v142 to KernelModelDriver10.0

**Comment by v-abdrahman** [comment]
github.com/ms-iot/samples link was given for HIDinjector as part of requirements and we have taken the code from the same location.
